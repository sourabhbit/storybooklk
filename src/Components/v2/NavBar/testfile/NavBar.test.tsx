import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import Navbar from '../index'

const props = {
	options: ['Your Picks', 'Discover', 'Categories', 'Gold', 'Collabs', 'Labs'],
}

describe('<Navbar />', () => {
	let _asFragment: any = null

	beforeEach(() => {
		const { asFragment } = render(<Navbar {...props} />)
		_asFragment = asFragment
	})

	afterAll(() => {
		_asFragment = null
	})

	it('Should render component', () => {
		const container = screen.getByTestId('navbar-wrapper')
		expect(container).toBeInTheDocument()
	})

	it('Should take a snapshot', () => {
		expect(_asFragment()).toMatchSnapshot()
	})
})
