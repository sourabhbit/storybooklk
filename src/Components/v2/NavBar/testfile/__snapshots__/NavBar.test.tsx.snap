// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> Should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL efRztj"
    data-testid="navbar-wrapper"
  >
    <div
      class="sc-gsDKAQ cmHcgF"
      data-testid="nav-button"
      id="id"
    >
      <span
        class="sc-dkPtRN eNtPVD"
        data-testid="nav-text"
      >
        Your Picks
      </span>
    </div>
    <div
      class="sc-gsDKAQ kjYOaa"
      data-testid="nav-button"
      id="id"
    >
      <span
        class="sc-dkPtRN fbKlCn"
        data-testid="nav-text"
      >
        Discover
      </span>
    </div>
    <div
      class="sc-gsDKAQ kjYOaa"
      data-testid="nav-button"
      id="id"
    >
      <span
        class="sc-dkPtRN fbKlCn"
        data-testid="nav-text"
      >
        Categories
      </span>
    </div>
    <div
      class="sc-gsDKAQ kjYOaa"
      data-testid="nav-button"
      id="id"
    >
      <span
        class="sc-dkPtRN fbKlCn"
        data-testid="nav-text"
      >
        Gold
      </span>
    </div>
    <div
      class="sc-gsDKAQ kjYOaa"
      data-testid="nav-button"
      id="id"
    >
      <span
        class="sc-dkPtRN fbKlCn"
        data-testid="nav-text"
      >
        Collabs
      </span>
    </div>
    <div
      class="sc-gsDKAQ kjYOaa"
      data-testid="nav-button"
      id="id"
    >
      <span
        class="sc-dkPtRN fbKlCn"
        data-testid="nav-text"
      >
        Labs
      </span>
    </div>
  </div>
</DocumentFragment>
`;
